pkg/apis/core/types_seed.go

```diff
@@ -105,7 +105,15 @@ type SeedBackup struct {
 	// SecretRef is a reference to a Secret object containing the cloud provider credentials for
 	// the object store where backups should be stored. It should have enough privileges to manipulate
 	// the objects as well as buckets.
+	// Deprecated: This field will be removed in a future version of Gardener. Use `CredentialsRef` instead.
```

Comment:
What is the future version? Please define it and add a TODO to follow-up: https://github.com/gardener/gardener/blob/master/docs/development/process.md#todo-statements

Reply:
I was looking at https://github.com/gardener/gardener/blob/3936598e7fee0e029b7bd93a88e5ca95cda81314/pkg/apis/core/types_shoot.go#L58-L59 as an example of similar change and was not aware about the introduced process. If I get it right, in this case the future version should be at least 9 released ahead in time?

Reply:
Since this affects only operators and not end-users, we could also go with the typical 3 releases, I guess?

Reply:
The field is set to be removed after v1.119.0 is released as this PR will be made available with v1.116.0 earliest.